src/main/java/org/folio/qm/converter/field/dto/BibliographicDataFieldConverter.java

```diff
@@ -13,16 +15,35 @@
 import org.springframework.stereotype.Component;
 
 @Component
+@RequiredArgsConstructor
 public class BibliographicDataFieldConverter extends CommonDataFieldConverter {
   private static final char AUTHORITY_ID_SUBFIELD_CODE = '9';
+  private static final char AUTHORITY_NATURAL_ID_SUBFIELD_CODE = '0';
+  private final LinksClient linksClient;
 
   @Override
   public FieldItem convert(DataField field, Leader leader) {
     var fieldItem = super.convert(field, leader);
     extractAuthorityId(field.getSubfields()).ifPresent(fieldItem::setAuthorityId);
+    setLinkingRuleId(fieldItem, field);
     return fieldItem;
   }
 
+  private void setLinkingRuleId(FieldItem fieldItem, DataField dataField) {
+    var subfields = dataField.getSubfields();
+    if (subfields.stream()
+      .anyMatch(subfield -> subfield.getCode() == AUTHORITY_ID_SUBFIELD_CODE)
+      && subfields.stream()
+      .anyMatch(subfield -> subfield.getCode() == AUTHORITY_NATURAL_ID_SUBFIELD_CODE)) {
+
+      var linkingRules = linksClient.fetchLinkingRules();
+      linkingRules.stream()
+        .filter(l -> l.getBibField().equals(dataField.getTag()))
+        .findFirst()
+        .ifPresent(l -> fieldItem.setLinkingRuleId(l.getId()));
+    }
+  }
+
```

Comment:
This logic doesn't need there as you have logic of assigning linkingRuleId in LinkService